### DISCUSSION TITLE
Script to quickly convert typst equations to latex

### DISCUSSION THREAD

**Discussion by tgilgien** [typst/typst#2815] [Show and tell]
Most markdown-based applications, including my favourite note-taking app, allow equations, but only in latex format. So after coping with very verbose syntax for a while, I made a little shell script so that I can type my equations with typst syntax and paste the equivalent latex equation (using [pandoc](https://pandoc.org/) behind the scene). It's available here: https://git.gilgien.net/theophile/tl/src/branch/main

It also allows abbreviations for long typst command I use often, for example `ubr()` for `underbrace()` or `..(x)` for `dot.double(x)`

**Comment by Quoteme** [comment]
code seems to be password protected. I cannot view it

**Reply by tgilgien** [reply]
Hi, thanks for pointing that out. Unfortunately, fixing the gitea configuration proved a little more difficult than anticipated; so in the meantime, here are the relevant files (GitHub doesn't allow to upload without file extensions):
[te.sh](https://github.com/user-attachments/files/25539962/te.sh)
[tl-backend.sh](https://github.com/user-attachments/files/25539965/tl-backend.sh)
[tl.sh](https://github.com/user-attachments/files/25539967/tl.sh)
[tli.sh](https://github.com/user-attachments/files/25539969/tli.sh)

Thanks for the interest!
